app/src/androidTest/java/com/duckduckgo/app/browser/favicon/DuckDuckGoFaviconManagerTest.kt

```diff
@@ -106,6 +107,16 @@ class DuckDuckGoFaviconManagerTest {
         verify(mockFaviconDownloader).getFaviconFromUrl(url.toUri().faviconLocation()!!)
     }
 
+    @Test
+    fun whenLoadToViewFromPersistedIfCannotFindFaviconThenLoadDefaultFaviconIntoView() = coroutineRule.runBlocking {
+        val view: ImageView = mock()
```

Comment:
Minor comment: we have some tests using `mock()` and other tests using `ImageView(InstrumentationRegistry.getInstrumentation().targetContext)`. No strong preference between any, maybe I would go fo `ImageView`. Are you ok using the same approach in each test?

Reply:
Oh, my bad, it was necessary to have the actual view before, now I can revert to `mock` all of them as it was before. I prob missed reverting one of them, will do

Reply:
@cmonfortep done. Resolving the comment :D